bcipy/display/demo/vep/demo_calibration_vep.py

```diff
@@ -1,61 +1,86 @@
+import logging
+import sys
+
 from psychopy import core, visual
 
+from bcipy.display import InformationProperties, VEPStimuliProperties
+from bcipy.display.components.layout import centered
+from bcipy.display.components.task_bar import CalibrationTaskBar
 from bcipy.display.paradigm.vep.display import VEPDisplay
+from bcipy.display.paradigm.vep.layout import BoxConfiguration
 from bcipy.helpers.clock import Clock
-from bcipy.display import InformationProperties, TaskDisplayProperties, VEPStimuliProperties
+
+root = logging.getLogger()
+root.setLevel(logging.DEBUG)
+handler = logging.StreamHandler(sys.stdout)
+handler.setLevel(logging.DEBUG)
+root.addHandler(handler)
 
 info = InformationProperties(
     info_color=['White'],
     info_pos=[(-.5, -.75)],
     info_height=[0.1],
-    info_font=['Consolas'],
+    info_font=['Arial'],
     info_text=['VEP Display Demo'],
 )
-task_display = TaskDisplayProperties(colors=['White'],
-                                     font='Consolas',
-                                     height=.1,
-                                     text='1/4')
 
 task_text = ['1/4', '2/4', '3/4', '4/4']
-task_color = [['white'], ['white'], ['white'], ['white']]
-num_boxes = 4
-start_positions_for_boxes = [(-.3, -.3), (.3, -.3), (.3, .3), (-.3, .3)]
+num_boxes = 6
 
-win = visual.Window(size=[700, 700], fullscr=False, screen=1, allowGUI=False,
-                    allowStencil=False, monitor='testMonitor', color='black',
-                    colorSpace='rgb', blendMode='avg',
+win = visual.Window(size=[700, 700],
+                    fullscr=False,
+                    screen=1,
+                    allowGUI=False,
+                    allowStencil=False,
+                    monitor='testMonitor',
+                    color='black',
+                    colorSpace='rgb',
+                    blendMode='avg',
                     waitBlanking=False,
                     winType='pyglet')
 win.recordFrameIntervals = True
 frameRate = win.getActualFrameRate()
 
 print(f'Monitor refresh rate: {frameRate} Hz')
 
-clock = core.Clock()
+box_colors = ['#00FF80', '#FFFFB3', '#CB99FF', '#FB8072', '#80B1D3', '#FF8232']
+stim_color = [[color] for i, color in enumerate(box_colors) if i < num_boxes]
+
+layout = centered(width_pct=0.95, height_pct=0.80)
+box_config = BoxConfiguration(layout, num_boxes=num_boxes)
+
 experiment_clock = Clock()
 len_stimuli = 10
 stimuli = VEPStimuliProperties(
-    stim_color=[['white'] * num_boxes],
-    stim_pos=start_positions_for_boxes,
+    stim_color=stim_color,
+    stim_pos=box_config.positions,
     stim_height=0.1,
-    stim_font='Consolas',
-    timing=(1, 0.5, 4),  # prompt, fixation, stimuli
+    stim_font='Arial',
+    timing=(1, 0.5, 2, 4),  # prompt, fixation, animation, stimuli

```

Comment:
Can we test these in a timing test? Only asking because I am worried there might be a lag as in Matrix case, where we set the stimulus duration as 0.25 but it ends up ~0.30s

Reply:
We probably want to create a timing task specific to VEP. I had to modify how I represented the checkerboards because it was taking more than the 1 second to display the sequence. However, I based this decision on the experiment clock timings. It would be better to confirm these values with the photodiode.

Reply:
This is more about how the `do_inquiry` method is implemented in the displays. We will time-test this code 👍🏼